plugins/inputs/aos/aos_test.go

```diff
@@ -44,17 +44,34 @@ func TestExtractProbeMessage(t *testing.T) {
 
 	tags := make(map[string]string)
 	tags["device"] = "probe_msg_1"
+	tags["probe_label"] = "probe_msg_label"
+
+	state := aos_streaming.RouteState_ROUTE_ADD
+	system_id := "1234567"
+	vni := uint32(5)
+	next_hop := "10.1.1.1"
+	rd := "100"
+	rt := "100"
+	evpn_type_3_route_event := &aos_streaming.EvpnType3RouteEvent{
```

Comment:
don't need to add test for evpn_type_5 route event?

Reply:
This test really only checks the tags. We do have other tests actually check the fields using `	acc.AssertContainsTaggedFields(t, "event_streaming", fields, tags)`

I can look at adding a test like this for probe message.

Reply:
I added a more comprehensive test for EvpnType3RouteEvent for probe messages.